client/src/components/Stars.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const { icons } = require('../img/icons.js');
+
+const Stars = ({ stars }) => {
+  const fullStar = <svg><path className="red" d={icons.star.full} /></svg>;
+  const emptyStar = <svg><path className="grey" d={icons.star.full} /></svg>;
+  const halfStar = (
+    <svg>
+      <path className="grey" d={icons.star.full} />
+      <path className="red" d={icons.star.half} />
+    </svg>
+  );
+  const allStars = [];
+  for (let i = 0; i < Math.floor(stars); i += 1) {
```

Comment:
I think eslint dislikes for-loops in general so I've been trying to use forEach or _.range where possible.  Not sure there's an easy replacement here but just a thought.

Reply:
My eslint isn't throwing an error for this, but I'll keep it in mind!